x-pack/plugins/event_log/README.md

```diff
@@ -274,21 +274,27 @@ PUT _ilm/policy/event_log_policy
       "hot": {                      
         "actions": {
           "rollover": {             
-            "max_size": "5GB",
+            "max_size": "50GB",
             "max_age": "30d"
           }
         }
+      },
+      "delete": {
+        "min_age": "90d",
+        "actions": {
+          "delete": {}
+        }
       }
     }
   }
 }
 ```
 
 This means that ILM would "rollover" the current index, say
-`.kibana-event-log-000001` by creating a new index `.kibana-event-log-000002`,
+`.kibana-event-log-8.0.0-000001` by creating a new index `.kibana-event-log-8.0.0-000002`,
 which would "inherit" everything from the index template, and then ILM will
 set the write index of the the alias to the new index.  This would happen
-when the original index grew past 5 GB, or was created more than 30 days ago.
+when the original index grew past 50 GB, or was created more than 30 days ago.
 
```

Comment:
Probably worth adding another sentence indicating the delete phase would then indicate that an index will be deleted 90 days after it has been rolled over.

Reply:
Doh! Yup you're right, will do

Reply:
Fixed in https://github.com/elastic/kibana/pull/64675/commits/1cd4171205c90afb03d8d51a6523cafa007502cd